### PR TITLE
feat: support cross origin request to tools and labs

### PR DESCRIPTION
**Pull request by dylandepass**
Fixes: #235 and #237

### CODE REVIEW

File: src/extension/app/aem-sidekick.css.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ export const style = css`
     pointer-events: none;
     z-index: 999999999999;
     font-weight: initial;
+    letter-spacing: initial;
```
Comment (dylandepass) [inline src/extension/app/aem-sidekick.css.js:25]: Noticed that sites modifying letter-spacing also had an impact on the sidekick.

File: src/extension/app/components/action-bar/picker/picker.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ export class Picker extends SPPicker {
       } else if (!this.pointerdownState) {
         // Prevent browser driven closure while opening the Picker
         // and the expected event series has not completed.
-        this.overlayElement.manuallyKeepOpen();
+        this.overlayElement?.manuallyKeepOpen();
```
Comment (dylandepass) [inline src/extension/app/components/action-bar/picker/picker.js:65]: This starting causing some issues with one of the tests

File: src/extension/auth.js [hunk 3]
```diff
@@ -49,9 +51,29 @@ export async function addAuthTokenHeaders() {
...
+
+      const corsRule = {
+        id: id++,
+        priority: 1,
+        action: {
+          type: 'modifyHeaders',
+          responseHeaders: [{
+            header: 'Access-Control-Allow-Origin',
+            operation: 'set',
+            value: '*',
```
Comment (rofe) [inline src/extension/auth.js:64]: Shouldn't this be set to `tools.aem.live` or `labs.aem.live` respectively?

**Review by rofe — APPROVED** [review]

File: src/extension/auth.js (hunk 3 above)
Comment (dylandepass) [reply src/extension/auth.js:64]: How would I know which one to set though? This is set at the time of login (on their aem.page/live or host).

Think I should create two rules then?

Comment (dylandepass) [reply src/extension/auth.js:64]: I think also `initiatorDomains` will have the same effect as being explicit with the ACAO header.

**Review by rofe — APPROVED** [review]

**Pull request merged**